### PR TITLE
Fix websockets in Safari

### PR DESCRIPTION
**Pull request by ryuuseijin**

### CODE REVIEW

**Comment by mayatron** [comment]
Yes, I need this fix also. 👍 🥇

File: src/util/websocket/browser_websocket.js [hunk 1]
```diff
@@ -4,7 +4,11 @@ var WS = global.MozWebSocket || global.WebSocket;
 
 module.exports = {
   create: function(url, protocols, options) {
-    if (typeof WS !== 'function') return null;
+    if (typeof WS !== 'function'
+        // In Safari it is an 'object'
```
Comment (ylin31) [inline src/util/websocket/browser_websocket.js]: Ran into this bug also. It's an object in Safari 9 (at least in 9.1.1), but in Safari 10 it's fixed and is 'function' as it should be.

**Comment by ylin31** [comment]
Awesome fix, please merge 🙌

**Comment by ryuuseijin** [comment]
I noticed that websockets aren't working properly in Safari 5 (Windows version), and we therefore shouldn't merge this. Safari<=9  will detect websockets correctly, so this fix is not really necessary.

**Pull request closed without merging**